goon/code/obj/machinery/bot/chefbot.dm

```diff
@@ -33,8 +33,7 @@
 	visible_message("<b>[src]</b> points at [target].")
 	if(istype(target, /atom))
 		var/D = new /obj/effect/decal/point(get_turf(target))
-		spawn(25)
-			qdel(D)
+		QDEL_IN(D, 25)
```

Comment:
```suggestion
		QDEL_IN(D, 2.5 SECONDS)
```

Reply:
just deleted this file, its not even compiled.